cogs/automation/intervally/daily_auto.py

```diff
@@ -50,6 +51,21 @@ async def create_daily_thread(self, daily_challenge_info):
         
         await thread.send(f"Daily Challenge - {display_date}", embed = embed)
         return
+    
+    async def remind_unverified(self):
+        guild = await self.client.fetch_guild(self.client.config['serverId'])
+        channel = await guild.fetch_channel("1090084731560927274")
+        random_prompts = [
+            ":blob_victory: :blob_victory: Hãy sử dụng lệnh </link:1206907242784235523> để kết nối tài khoản Leetcode của mình và khám phá những tính năng độc đáo của chúng mình nhé!",
+            "Hãy </link:1206907242784235523> tài khoản bằng LLC Assistant để luyện tập cùng chúng mình nào! :blob_taco: :blob_taco:",
+            "Ồ, chào bạn. Có vẻ bạn quên </link:1206907242784235523> tài khoản Leetcode nè! :blob_maman: :blob_maman: :blob_taco:",
+            ":eyes: :eyes: Bạn có biết, </link:1206907242784235523> tài khoản Leetcode với chúng mình sẽ giúp bạn đạt mục tiêu dễ dàng hơn?",
+            ":100: :100: :100: :100: :100: Đã có trên 200 người </link:1206907242784235523> tài khoản với chúng mình. Một phần không nhỏ đã đạt được mục tiêu 500 bài. Liệu bạn có phải người tiếp theo? :blob_taco: :blob_taco:",
+            ":beers: :game_die: </link:1206907242784235523> tài khoản, tham gia cùng server, để không bỏ lỡ thông báo mới nhất về các buổi offline nhé! :blob_taco: :blob_taco: :blob_taco: :blob_taco:",
+            ":eyes: :blob_taco: :blob_taco: Chúng mình có bí kíp code khủng mà vẫn được chạm cỏ thường xuyên. :eyes: :eyes: -> </link:1206907242784235523>",
```

Comment:
Either directly replace animated emojis with `<a:{name}:{id}>` (e.g. `<a:blob_taco:1086131891796709457>`) or add them to `utils.asset` for reusing.